scripts/run-lint.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
```

Comment:
Exit on error?
```suggestion
set -e

```

Reply:
turns out we can't, since exiting within the inner function causes an error to be returned from the outer function